docs/src/componentDocs/DrawerFooter/examples/ComplexDrawerFooterExample.tsx

```diff
@@ -15,12 +15,18 @@ export const ComplexDrawerFooterExample = (): JSX.Element => (
                 </DrawerNavGroup>
             </DrawerBody>
             <DrawerFooter sx={{ p: 2 }}>
-                <Box sx={{ mb: 1 }}>
-                    <Box>
-                        <FormHelperText>v2.4.0</FormHelperText>
-                    </Box>
-                    <Box>
-                        <FormHelperText>10:33:05 03/12/22</FormHelperText>
+                <Box>
+                    <Box sx={{ mb: 1 }}>
+                        <Box>
+                            <Typography variant="caption" color={colors.BLUIColors.gray[200]}>
+                                v2.4.0
+                            </Typography>
+                        </Box>
+                        <Box>
+                            <Typography variant="caption" color={colors.BLUIColors.gray[200]}>
+                                10:33:05 03/12/22
+                            </Typography>
+                        </Box>
                     </Box>
                     <Box sx={{ display: 'flex', justifyContent: 'space-between', alignItems: 'center', mt: 1 }}>
                         <Box sx={{ width: 83 }} component="img" src={EatonFooterLogoLight} />
```

Comment:
Can we add '../images/EatonLogoLight.png' instead of src={EatonFooterLogoLight} ?